app/main/views/brief_responses.py

```diff
@@ -214,6 +214,13 @@ def list_brief_responses():
     if brief_id is not None:
         brief_responses = brief_responses.filter(BriefResponse.brief_id == brief_id)
 
+    if request.args.get('framework_slug'):
+        brief_responses = brief_responses.join(BriefResponse.brief).join(Brief.framework).filter(
+            Brief.framework.has(Framework.slug.in_(
+                framework_slug.strip() for framework_slug in request.args["framework_slug"].split(",")
```

Comment:
I'm not sure about using `framework_slug` here, I can see the point about clarity, but there's a counterargument about consistency with usage elsewhere, and elsewhere we have `framework` arg for a list of framework slugs.

It's probably OK I guess, not a deal-breaker if you prefer it this way.